.github/workflows/tagged-release.yml

```diff
@@ -15,16 +15,19 @@ jobs:
     steps:
       - name: Create Release
         id: create_release
-        uses: actions/create-release@v1
```

Comment:
Archived at 3 years ago.

https://github.com/softprops/action-gh-release has almost the same interface(inputs/outputs).